src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import {Dimensions} from 'react-native';
+
+import lodashGet from 'lodash/get';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import PopoverWithMeasuredContent from '../../../../components/PopoverWithMeasuredContent';
+
+import BaseReactionList from './BaseReactionList';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class PopoverReactionList extends React.Component {
```

Comment:
I think it could work, but I have some concerns about how often we're calling the component since it's a popup and the complexity of the difference in components.

Reply:
no, I didn't, I actually considered making a general component for popups, but decided against it due to the many differences under the hood. But I totally agree with you - it'd be great to have a unified component in the future. So let me try to do this